cli/integration_tests/prune/composable-config.t

```diff
@@ -0,0 +1,26 @@
+Setup
+  $ . ${TESTDIR}/../setup.sh
+  $ . ${TESTDIR}/setup.sh $(pwd)
+
+Make sure that the internal util package is part of the prune output
+  $ ${TURBO} prune --scope=docs
+  Generating pruned monorepo for docs in .*\/out (re)
+   - Added docs
+   - Added shared
+   - Added util
+  $ cd out && ${TURBO} run new-task
+   WARNING  cannot find a .git folder. Falling back to manual file hashing (which may be slower). If you are running this build in a pruned directory, you can ignore this message. Otherwise, please initialize a git repository in the root of your monorepo
+  \xe2\x80\xa2 Packages in scope: docs, shared, util (esc)
+  \xe2\x80\xa2 Running new-task in 3 packages (esc)
+  \xe2\x80\xa2 Remote caching disabled (esc)
+  docs:new-task: cache miss, executing 89b0cf4ede0c4ae5
+  docs:new-task: 
+  docs:new-task: > docs@ new-task .*out/apps/docs (re)
```

Comment:
this test repo uses pnpm, and pnpm does this thing where it prints out the directory where the task was run from as part of the logging. npm doesn't do this.